src/hooks/JB721Delegate/HasNftRewards.ts

```diff
@@ -22,19 +21,13 @@ export function useHasNftRewards(): { value: boolean; loading: boolean } {
     fundingCycleMetadata,
     loading: { fundingCycleLoading },
   } = useContext(V2V3ProjectContext)
-  const { value: supportsV1Interface, loading: v1Loading } =
-    useIsJB721DelegateV1({
-      dataSourceAddress: fundingCycleMetadata?.dataSource,
-    })
-  const { value: supportsV1_1Interface, loading: v1_1Loading } =
-    useIsJB721DelegateV1_1({
-      dataSourceAddress: fundingCycleMetadata?.dataSource,
-    })
```

Comment:
yeah so...this was a big issue lol. we were calling out to infura every time this hook is called, which is a lot.

Reply:
Gotcha, good catch.